src/python/engagement-creator/engagement_creator/lib/engagement_creator.py

```diff
@@ -74,21 +74,20 @@ def risk_node(self, analyzer_name: str) -> None:
         )
 
 
-def parse_s3_event(s3: S3ServiceResource, event: Any) -> bytes:
-    # Retrieve body of sns message
-    # Decode json body of sns message
-    LOGGER.debug("event is {}".format(event))
-
-    bucket = event["s3"]["bucket"]["name"]
-    key = event["s3"]["object"]["key"]
+def parse_s3_event(s3: S3ServiceResource, event: S3PutRecordDict) -> str:
+    try:
+        bucket = event["s3"]["bucket"]["name"]
+        key = event["s3"]["object"]["key"]
+        key = key.replace("%3D", "=")
```

Comment:
this is super odd to me, but 
- it was there in the previous version of download_s3_file
- it _did_ solve the bug i saw
- theres probably a better way to de-urlencode it but whatever man  it WORKS